tests/kernel/fatal/src/main.c

```diff
@@ -115,15 +115,30 @@ void alt_thread3(void)
 	irq_unlock(key);
 }
 
+#ifdef CONFIG_STACK_SENTINEL
 void blow_up_stack(void)
 {
 	char buf[OVERFLOW_STACKSIZE];
 
 	TC_PRINT("posting %zu bytes of junk to stack...\n", sizeof(buf));
 	(void)memset(buf, 0xbb, sizeof(buf));
 }
+#else
+/* Better implementation IMO, but stack sentinel doesn't catch it in time
```

Comment:
That should work. Myself, I had tried a Fibonacci sequence implementation with recursive calling.

Reply:
Could you remove the comment, so it does not reflect your personal opinion? :)

Reply:
I wrote both implementations, but sure